examples/rtmicrophone.html

```diff
@@ -0,0 +1,107 @@
+<html>
```

Comment:
It might make sense to roll this processStream example into the existing example examples/microphone.html, since that one does the same things otherwise as this example does. But that example is a little broken right now because the control icons are not showing up, which looks like an issue with the latest dreem text changes, and I'm looking into it now.